### PR TITLE
GSREN3D-307 : Fix bugs parking, click station & img service

### PR DESCRIPTION
**Pull request by toni-garcia-camptocamp**
GSREN3D-307 : Fix bugs parking, click station & img service

### JIRA issue

https://jira.camptocamp.com/browse/GSREN3D-307

### Description

Fix bug : fetch parkings by station instead of fetch parking by line
Fix bug: img service online are not working
Fix bug: click on another station on station view doesn't work

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -35,7 +35,7 @@ const isPhotoGalleryVisible = computed(() => {
   <main class="h-screen flex">
     <aside class="z-10 absolute">
       <SidePanel v-show="panelStore.isPlanningViewShown === false">
-        <RouterView />
+        <RouterView :key="$route.fullPath" />
```
Comment (ismailsunni) [inline src/App.vue:38]: what is this for?

**Review by ismailsunni — APPROVED** [review]
LGTM

File: src/App.vue (hunk 1 above)
Comment (toni-garcia-camptocamp) [reply src/App.vue:38]: I don't know why but without the key on RouteView i had one bug : 
When i clicked on another station on station page, the url change but not the page. With key on RouterView, the url change and the page too.

File: src/interactions/clickAndMoveInteraction.ts [hunk 2]
```diff
@@ -35,11 +35,11 @@ class mapClickAndMoveInteraction extends AbstractInteraction {
     const stationName = feature?.get('nom')
     if (event.type & EventType.CLICK) {
       const viewStore = useViewsStore()
-      if (viewStore.currentView == viewList.line) {
+      if ([viewList.line, viewList.station].includes(viewStore.currentView)) {
         const lineViewStore = useLineViewsStore()
         const lineNumber = lineViewStore.selectedLine
         const stationId = feature?.get('id')
-        router.push(`/line/${lineNumber}/station/${stationId}`)
+        router.push({ path: `/line/${lineNumber}/station/${stationId}` })
```
[the remark this answers is not in the document]
Comment (toni-garcia-camptocamp) [reply src/interactions/clickAndMoveInteraction.ts:42]: It's the same . I had a bug and this was an attempt to fix it. But the problem did not come from here. I'll put the call back like before

**Pull request merged**